Finishing touches for the GUI

skaupper opened the pull request:
Reset the GUI as soon as the app disconnects from the simulator
Log only the top most exception message

skaupper committed:
reset the GUI as soon as the app disconnects from the simulator
log only the top most exception message

pagdot commented on src/main/java/at/fhhagenberg/esd/sqe/ws20/gui/ECCController.java:
```diff
@@ -217,6 +215,21 @@ private void connect(boolean initial) {
+                    floor.requestUp.set(false);
+                    floor.stopRequest.set(false);
+                }
+                targetFloor.set(0);
```
I think setting to `-1` should set it to *none*

skaupper committed:
do not select any target floor when disconnected

skaupper committed:
print a log message if the app successfully connects to the simulator

skaupper committed:
fix ErrorTests

pagdot committed:
fix direction

pagdot committed:
Refactor update method

- Update floors even when no elevator is selected
- Make sure only one error message is logged

pagdot committed:
reduce complexity
